dist/index.js

```diff
@@ -8095,7 +8101,7 @@ function fetch(url, opts) {
 				// HTTP fetch step 5.5
 				switch (request.redirect) {
 					case 'error':
-						reject(new FetchError(`redirect mode is set to error: ${request.url}`, 'no-redirect'));
+						reject(new FetchError(`uri requested responds with a redirect, redirect mode is set to error: ${request.url}`, 'no-redirect'));
```

Comment:
```suggestion
						reject(new FetchError(`Requested URI responds with a redirect, redirect mode is set to error: ${request.url}`, 'no-redirect'));
```

Reply:
This is part of node-fetch, is just there because is the build.

Reply:
ah, my bad